src/Components/ValidationMessage.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import { ValidationState } from '../Models/PatientFieldData';
+
+function ValidationMessage({ className, validationState }) {
```

Comment:
I added className as a prop to ValidationMessage, and use the classnames library to merge that in below, so that there's more flexibility how they can be styled...